### ISSUE TITLE
mantichora broken on Windows with release 0.9.8?

### ISSUE THREAD

**Issue by domengorjup** [alphatwirl/mantichora#5]
First of all I would like to thank you for your work on this awesome package.

With the 0.9.8 release (and more specifically, commit 0216c89c9798892d6a7d79217c11ce44ef82d8d2), manthicora has stopped working for me on Windows. As far as I understand, `fork` is not available with multiprocessing on Windows.

Is there a way to possibly revert to previous behaviour on Windows in a future release?

**Comment by TaiSakuma** [comment]
Thank you for letting me know.

I will see if I can revert the default to `spawn` on Windows.

Incidentally, in the latest version (0.10.0), you can choose to use`spawn` by the option `mp_start_method` as described in a [section of README.md](https://github.com/alphatwirl/mantichora/tree/v0.10.0#start-method-of-multiprocessing).

```python
mantichora(mp_start_method='spawn')
```

However, this still might not work on Windows. I don't have access to a Windows machine and cannot try it at the moment.

**Comment by domengorjup** [comment]
I don't think that would work, since the error is thrown already when importing mantichora. Here is more information I hope you might find helpful:

```
(venv) λ python
Python 3.7.1 (default, Oct 23 2018, 22:56:47) [MSC v.1912 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import mantichora
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\test\venv\lib\site-packages\mantichora\__init__.py", line 2, in <module>
    from .main import mantichora
  File "C:\test\venv\lib\site-packages\mantichora\main.py", line 4, in <module>
    from .hubmp import MultiprocessingHub
  File "C:\test\venv\lib\site-packages\mantichora\hubmp.py", line 272, in <module>
    ctx_fork = multiprocessing.get_context('fork')
  File "c:\programdata\anaconda3\Lib\multiprocessing\context.py", line 238, in get_context
    return super().get_context(method)
  File "c:\programdata\anaconda3\Lib\multiprocessing\context.py", line 192, in get_context
    raise ValueError('cannot find context for %r' % method) from None
ValueError: cannot find context for 'fork'
```

The issue seems to be in the following line:
https://github.com/alphatwirl/mantichora/blob/c0b32d04cc948fc03ec0a360ed1c26a325d0a8f1/mantichora/hubmp.py#L272
as this code fails on Windows.

This is for version 0.10.0 of mantichora.

**Comment by TaiSakuma** [comment]
Thank you for checking. I will fix it when I can.

**Comment by TaiSakuma** [comment]
This issue has been open for a while. I finally had a chance to work on it.
I released [0.12.0](https://pypi.org/project/mantichora/0.12.0/).
[It works on Windows in GitHub Actions.](https://github.com/alphatwirl/mantichora/actions/runs/845811367)
